mne_bids/commands/mne_bids_raw_to_bids.py

```diff
@@ -4,7 +4,7 @@
 """Command line interface for mne_bids.
 
 example usage:  $ mne_bids raw_to_bids --subject_id sub01 --task rest
---raw_file data.edf --output_path new_path
+--raw_file data.edf --kind eeg --output_path new_path
```

Comment:
here you write eeg. Then below in help you don't specify the possible values and in the function doctring options are marked as "meg" or "ieeg".

can you clarify at all places?

Reply:
fixed, the example i chose before used `.edf` and matched the `kind` argument to it. since `eeg` isn't supported yet, I changed it to `.fif`